File: Backend/api/app_server/models/customer.js
const mongoose = require('mongoose');
const bcrypt = require('bcrypt-nodejs');

// Customer Schema th

const customerSchema = mongoose.Schema({
	name:{
		type: String,
	},
	email:{
		type: String,
	},
	contact:{
		type: String
	},
	picture:{
		type: String,
	},
	password:{
		type: String
	},
	role:
    {
        type:Number,
        default:2
	},
	state:
    {
		type: String,
		default:"approved"
    }
});
customerSchema.methods.hashPassword = function(password){
	return bcrypt.hashSync(password,bcrypt.genSaltSync(10))
}
customerSchema.methods.comparePassword = function(password,hash){
	return bcrypt.compareSync(password,hash)
}
 

const customer = module.exports = mongoose.model('Customer', customerSchema);

File: Android/sportx/app/src/main/java/com/sportx/pk/Models/Customer.java
package com.sportx.pk.Models;

public class Customer {

    private String name;
    private String email;
    private String picture;
    private String contact;
    private String password;


    public Customer( String name, String email, String picture, String contact,  String password) {

        this.name = name;
        this.email = email;
        this.picture = picture;
        this.contact = contact;
        this.password = password;

    }


    public String getCustomerName() {
        return name;
    }

    public String getCustomerEmail() {
        return email;
    }

    public String getCustomerContact() {
        return contact;
    }

    public String getCustomerPicture() {
        return picture;
    }

    public String getCustomerPassword() {
        return password;
    }

}
